src/storage.ts

```diff
@@ -488,6 +502,24 @@ export class Storage {
     }
   }
 
+  /**
+   * Migrate the session token file from "session_token" to "session", if needed.
+   */
+  public async migrateSessionToken(label: string) {
+    const oldTokenPath = this.getLegacySessionTokenPath(label)
+    try {
+      await fs.stat(oldTokenPath)
+      const newTokenPath = this.getSessionTokenPath(label)
+      await fs.rename(oldTokenPath, newTokenPath)
```

Comment:
Not a blocker at all, just wanted to mention that `stat` is often an anti-pattern; because `rename` can also throw `ENOENT` and there is technically no guarantee the file still exists between the `stat` and `rename`, it is always necessary to check the error of `rename` anyway, so might as well do it with one call.

Reply:
Right, that makes sense. Removed the `stat`.